backend/services/z_notion.js

```diff
@@ -44,7 +60,7 @@ module.exports = async function getMembers() {
     //console.log(results);
 
     const members = results.map(page => {
-        console.log(page);
+        //console.log(page);
```

Comment:
In production (eventually), make sure to remove console logs, not just comment them out